tor/cmd_onion.go

```diff
@@ -253,12 +261,19 @@ func (c *Controller) AddOnion(cfg AddOnionConfig) (*OnionAddr, error) {
 	c.activeServiceID = serviceID
 	log.Debugf("serviceID:%s added to tor controller", serviceID)
 
+	// We return the onion address private key if requested
```

Comment:
Since `returnKey` is false if the private key is on the disk in plain text, does it really make sense to add this parameter? Can't we just always set the key in memory?

Reply:
Makes sense to me